src/views/process-panel/process-information-card.tsx

```diff
@@ -94,20 +94,25 @@ export const ProcessInformationCard = withStyles(styles, { withTheme: true })(
                     </div>
                 }
                 title={
-                    <Tooltip title={process.containerRequest.name} placement="bottom-start" color='inherit'>
-                        <Typography noWrap variant="title">
-                           {process.containerRequest.name}
+                    <Tooltip title={process.containerRequest.name} placement="bottom-start">
+                        <Typography noWrap variant="title" color='inherit'>
+                            {process.containerRequest.name}
                         </Typography>
                     </Tooltip>
                 }
-                subheader={process.containerRequest.description} />
+                subheader={
+                    <Tooltip title={process.containerRequest.description || '(no-description)'} placement="bottom-start">
```

Comment:
You could create a method and return description or default value. 
You duplicate a code here.